test/elixir_analyzer/test_suite/rpg_character_sheet_test.exs

```diff
@@ -145,6 +145,39 @@ defmodule ElixirAnalyzer.TestSuite.RpgCharacterSheetTest do
           |> IO.gets()
           |> String.trim()
         end
+      end,
+      defmodule RPG.CharacterSheet do
+        import IO
```

Comment:
Should we also add a test for `newsletter` that uses imports?

Reply:
Sure! I'll add one tomorrow.